atlp-rwanda/hackers-ec-Fe
# #187418997 Setting up TailwindCss styling Environment

#### What does this PR do?

Setting up Tailwind CSS styling environment

#### Description of Task to be completed?

- installing tailwind package
- configuring tailwind.config
- setting up global colors and screens

#### How should this be manually tested?
- Clone this repository
- Run npm install to install all packages
- run npm run dev 
- Start styling and observe changes

#### What are the relevant pivotal tracker/Trello stories?

#187418997
